docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -29,7 +29,10 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
 - **ORDER BY**
 
 The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
```

Comment:
For the next minor release, we're gonna disable this feature by default in OSS. It's controlled via dynamic config.
cc: @yiminc

Reply:
@rodrigozhou @yiminc Can you make a note wherever you are tracking that release to update the docs on this point when it becomes relevant to do so?